crates/indexer/src/runtime.rs

```diff
@@ -238,6 +241,18 @@ pub async fn run_forever(config: IndexerConfig, context: Arc<Context>) {
             }
         };
 
+        tailer
+            .update_last_processed_version(&processor_name, processing_result.end_version)
+            .unwrap_or_else(|e| {
+                error!(
+                    processor_name = processor_name,
+                    end_version = processing_result.end_version,
+                    error = format!("{:?}", e),
```

Comment:
`error =? e` is the same without the format.

Reply:
Oh cool.